### PR TITLE
Header styling

### PR DESCRIPTION
**Pull request by andrewMacmurray**
header now has styling, logo and shopping cart icon

### CODE REVIEW

**Comment by katbow** [comment]
@andrewMacmurray I fixed the merge conflict in this, however the logo image isn't showing up on my computer (both before & after merge so I don't think anything necessary was deleted). Do you know why that might be happening?

**Comment by andrewMacmurray** [comment]
@katbow good job :+1: I think the logo probably isn't loading because the src is hard coded in (are you running it from build or dev?)

**Comment by katbow** [comment]
I ran it from build

**Comment by katbow** [comment]
P.S. I love the little shopping cart :heart: . And everything else of course :smile:

File: frontend/src/js/Header/index.jsx [hunk 1]
```diff
@@ -7,8 +7,12 @@ class Header extends React.Component {
   render () {
     return (
       <div>
-        <Link to='/'><h1>Amazon 2.0 Home Page</h1></Link>
-        <header style={styles}>
+        <header className='container' style={styles}>
+          <Link to='/'>
+            <div className='logo-container'>
+              <img src='../../../public/img/shamazon-logo.png'></img>
```
Comment (andrewMacmurray) [inline frontend/src/js/Header/index.jsx:13]: @katbow if you want to run this from build replace the src with 'img/shamazong-logo.png'

**Comment by andrewMacmurray** [comment]
@katbow  :cupid:

File: frontend/src/js/Header/index.jsx (hunk 1 above)
Comment (katbow) [inline frontend/src/js/Header/index.jsx]: /play tada

**Comment by katbow** [comment]
@andrewMacmurray is this all in PR #136 in the success-message branch?

**Pull request closed without merging**